apps/protailwind/src/lib/exercises.ts

```diff
@@ -42,6 +43,7 @@ export const ExerciseSchema = z
         _key: z.string(),
         videoResourceId: z.nullable(z.string()).optional(),
         transcript: z.nullable(z.any().array()).optional(),
+        stackblitz: z.nullable(z.string()).optional(),
```

Comment:
my approach to this, instead of polluting our entire space with resource types that are never used is to treat resource loading as a separate concern from an "exercise" (lesson, generally) and to the extent possible focus on the lesson type

because of the nature of Lesson resources they can be loaded clientsode (trpc) like this: 

https://github.com/skillrecordings/products/blob/main/apps/total-typescript/src/trpc/routers/stackblitz-resource.ts

Now, we can load a given Lesson's resources with the lesson ID and the type/implementation can be LOCAL to the project and we don't have to do stuff like the zod changes here polluting this project with stuff that will never be used